src/content/testing/common-errors.md

```diff
@@ -475,43 +496,49 @@ framework for every frame, for example, during an animation.
 
 **How to fix it?**
 
-One way to avoid this error is to use the `Navigator` API
-to trigger the dialog as a route. In the following example,
-there are two pages. The second page has a
-dialog to be displayed upon entry.
-When the user requests the second page by
-clicking a button on the first page,
-the `Navigator` pushes two routes–one
-for the second page and another for the dialog.
+One way to avoid this error is instruct flutter to finish rendering the page
+before implementing the showDialog. This can be done by using
+addPostFrameCallback() method. The following code illustrates this on the broken example just given:
```

Comment:
Generally we want files to have lines of <= 80 characters. ([more info here](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks))

Reply:
I split line 501 into two in my fork of the code, but I'm not sure if this change makes its way automatically into the PR, or whether I need to do something to update it.

Reply:
It looks like this PR is using the [`554richard:patch-1`](https://github.com/554richard/website/tree/patch-1) branch. If you push changes to that branch, they'll show up in this pull request!